score-xml/src/test/java/io/cloudslang/content/xml/actions/EditXmlTest.java

```diff
@@ -0,0 +1,418 @@
+package io.cloudslang.content.xml.actions;
+
+import org.apache.xml.serialize.OutputFormat;
+import org.apache.xml.serialize.XMLSerializer;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.w3c.dom.Document;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.*;
+import java.util.Map;
+
+/**
+ * Created by moldovas on 6/22/2016.
+ */
+public class EditXmlTest {
+
+    private static final String EMPTY_STRING = "";
+    private static final String DELETE = "delete";
+    private static final String RENAME = "rename";
+    private static final String INSERT = "insert";
+    private static final String APPEND = "append";
+    private static final String SUBNODE = "subnode";
+    private static final String MOVE = "move";
+    private static final String UPDATE = "update";
+    private static final String ELEM = "elem";
+    private static final String ATTR = "attr";
+    private static final String TEXT = "text";
+    private static final String STRING_XML = "<?xml version=\"1.0\" ?>\n" +
```

Comment:
@tethryus ... neah... you should write it from a resource bundle file... check out the rnd/openstack repo to see how